app/views/middleware_topology/show.html.haml

```diff
@@ -10,7 +10,7 @@
         %svg.kube-topology
           %g.MiddlewareServer{:transform => "translate(21, 21)"}
             %circle{:r => "17"}
-            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/100/middleware_server.png", :y => "-9"}
+            %image{:height => "20", :width => "20", :x => "-9", "xlink:href" => "/assets/svg/vendor-wildfly.svg", :y => "-9"}
```

Comment:
@pilhuhn @himdel I'm having some thoughts putting the icons data on the $scope given that #7463 was merged, and then maybe we can leverage that data here generically that will solve the path problem. not sure yet how to retrieve it nicely in haml, but the data can be made pretty easily available on $scope.